pkgs/applications/misc/electron-cash/default.nix

```diff
@@ -3,15 +3,23 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "electron-cash";
-  version = "4.2.5";
+  version = "4.2.6";
 
   src = fetchFromGitHub {
     owner = "Electron-Cash";
     repo = "Electron-Cash";
     rev = version;
-    sha256 = "sha256-ALIrNnhpX46xdQdfJdx/9e/QtdyBEgi5xLrbuOBJR7o=";
+    sha256 = "sha256-/5b9V1m7hBI6qsLr1/nE9bWj2SPX9SmdlzzFqS3Nc6Q=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "relax-qdarkstyle-version-bound.patch";
+      url = "https://github.com/Electron-Cash/Electron-Cash/pull/2396.patch";
```

Comment:
Please don't use the PR diff URL. They are not deterministic and may cause build failure later. Use commit diff URL instead:
- `https://github.com/Electron-Cash/Electron-Cash/commit/086c7f797f60e9f7c84bc622d867c98cb3b743ee.patch`
- `https://github.com/Electron-Cash/Electron-Cash/commit/8d0aa028e1665196e6c8643eae1fbf9fae33e9f4.patch`

Also, let's add a comment for the PR URL before `fetchpatch`.